src/Microsoft.Health.Fhir.TemplateManagement/Models/TemplateLayer.cs

```diff
@@ -17,21 +18,27 @@ public class TemplateLayer : OCIArtifactLayer
     {
         public Dictionary<string, Template> TemplateContent { get; set; }
 
-        public static TemplateLayer ReadFromFile(string filePath)
+        public static TemplateLayer ReadFromEmbeddedResource()
```

Comment:
i feel we can keep the original api and also add a new one ReadFromStream().
both ReadFromFile and ReadFromEmbeddedResource will call ReadFromStream.

Reply:
Sure. Have added the ReadFromFile() api back.